exporters/write_buffer.py

```diff
@@ -163,9 +163,40 @@ def _create_buffer_file(self, file_name=None):
                           self.compression_format, file_name=file_name)
 
 
+class FilebasedGroupingBufferFilesTracker(GroupingBufferFilesTracker):
```

Comment:
Why are you copying this class code here? it is already in exporters/filebase_base_writer.py